docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -130,14 +132,14 @@ In the case of starting the removal process, the topic's description in the reas
 Chairperson
 ------------
 
-The chairperson election will happen once a year around the time of
-Ansible Fest. If the current chairperson has to step down early, the election happens immediately.
+The chairperson election happens once a year at the time the Committee agrees on by voting in a dedicated forum thread.
```

Comment:
Don't think we've done this so far, do we need this line?

Reply:
@gundalow it's a good point worth discussing!
I've already started the vote though. I don't think this can be discussed quickly and i think it should be done thoroughly.
Let's maybe postpone this question to another pass?